configs/components/rubygem-puppet.rb

```diff
@@ -1,13 +1,13 @@
 component 'rubygem-puppet' do |pkg, settings, platform|
   # Projects may define a :rubygem_puppet_version setting, or we use 6.20.0 by default:
-  version = settings[:rubygem_puppet_version] || '6.20.0'
```

Comment:
Hmm, any reason why we're not calculating this from `rubygem_puppet_version` ?

Reply:
I figured we would do that work when we pull in the puppet7 in our components ticket. same as the gemspec change in the ace repo.

Reply:
oh, hmm seems we need a ticket for that https://tickets.puppetlabs.com/browse/PE-31684

Reply:
oh, nevermind, found it https://tickets.puppetlabs.com/browse/PE-31696

Reply:
as part of that ticket we will move to setting :rubygem_puppet_version in the individual bolt-runtime project definitions.